### PR TITLE
feature/LIN-167-LIN-168/integrate-Heap-and-Hotjar-to-schedule-link-app

### PR DESCRIPTION
**Pull request by trungducng0410**
### Issue ticket number and link
[LIN-167]
[LIN-168]
### Describe main changes
Screenshot 2024-08-27 at 09 22 25

### Checklist in Local 
- [ ] I have extract component business logic into custom hook
- [ ] I have added unit tests
- [ ] I have integrated with backend api and done a e2e test locally
- [ ] I have updated the graphql query types

### Checklist in Dev
- [ ] I have deployed to dev
- [ ] I have integrated with backend api and done a e2e test

### Checklist before requesting a review
- [ ] I have done the **code format** using `pnpm lint:fix` and `pnpm format:fix`
- [ ] I have **rebase squash** current branch
- [ ] I have confirmed **Naming Convention** with team if needed.
- [ ] I have done a **self review** in GitHub.

### Checklist in Qa Environment
- [ ] I have deployed to qa and let QA team do a test
- [ ] I have fix the QA bugs after QA team finish testing
- [ ] I have put the screenshot for Qa environment integration test below:

### Checklist in Sandbox Environment
- [ ] I have deployed to sandbox and let QA team do test
- [ ] I have fix the Sandbox Bugs after QA team finish testing
- [ ] I have put the screenshot for Qa environment integration test below:

### Checklist after Merge
- [ ] I have known the dependencies of release deployment of current PR.
- [ ] I have reviewed the release pipe from team lead to secure the production deployment.
- [ ] I have update the Jira tickets status after all task has been finished.

[LIN-167]: https://predictivehire.atlassian.net/browse/LIN-167?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LIN-168]: https://predictivehire.atlassian.net/browse/LIN-168?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -23,20 +23,10 @@ steps:
   - wait
 
   - block: "Deploy to DEV"
-    branches:
-      - "develop"
-      - "bugfix/*"
-      - "hotfix/*"
-      - "feature/*"
```
Comment (bennyli519) [inline .buildkite/pipeline.yml:30]: may i know why we need to remove these?

Comment (trungducng0410) [reply .buildkite/pipeline.yml:30]: I feel like @yingzhi-lu want to use whitelist for SANDBOX/PROD only, for DEV/QA we could allow any kind of branch name to be deployed. Pls correct me if I'm wrong @yingzhi-lu

Comment (yingzhi-lu) [reply .buildkite/pipeline.yml:30]: Hi @trungducng0410 it was my testing branch as I want to push my feature branch to QA so I removed them for convenience, could you revert it back? Thanks

**Review by bennyli519 — APPROVED** [review]
LGTM

**Pull request merged**